src/widgets/VariantDivisionDistributionChart.tsx

```diff
@@ -32,70 +35,216 @@ const countriesWithMaps = [
   'Botswana',
 ];
 
+const TableHeader = styled.th`
+  font-weight: bolder;
+  margin-top: 10px;
+  &:hover,
+  &:focus {
+    color: #0276fd;
+    cursor: pointer;
+  }
+`;
+
+const sortOptions = ['division', 'count', 'prevalence'] as const;
+type SortOptions = typeof sortOptions[number];
+
+interface GeoSummary {
+  division: string;
+  count: number;
+  prevalence: number;
+}
+
+const sortGeoTable = (entries: GeoSummary[], sortOption: SortOptions, ifAscendic: boolean): GeoSummary[] => {
+  switch (sortOption) {
+    case 'count':
+      return ifAscendic
+        ? entries.sort((a, b) => a.count - b.count)
+        : entries.sort((a, b) => b.count - a.count);
+    case 'prevalence':
+      return ifAscendic
+        ? entries.sort((a, b) => a.prevalence - b.prevalence)
+        : entries.sort((a, b) => b.prevalence - a.prevalence);
+    case 'division':
+      return ifAscendic
+        ? entries.sort((a, b) => (a.division > b.division ? 1 : b.division > a.division ? -1 : 0))
+        : entries.sort((a, b) => (b.division > a.division ? 1 : a.division > b.division ? -1 : 0));
+  }
+};
+
 export const VariantDivisionDistributionChart = ({
   variantSampleSet,
   wholeSampleSet,
 }: VariantDivisionDistributionChartProps) => {
+  const [geoOption, setGeoOption] = useState<string>('Countries');
+  const [sortOption, setSortOption] = useState<SortOptions>('division');
```

Comment:
It is strange to use lower case as internal value for one variable and upper case for another. Please try to be consistent (and use lower case).